basics/scenarios.md

```diff
@@ -10,8 +10,8 @@ In this case, we will use just Perceval, as a Python module, to retrieve informa
 
 As shown in the figure above, data retrieved by Perceval is consumed by a Python script, which produces, for example, a summary of the data in the repository. In this simple scenario, most of the components in GrimoireLab are not needed, and the setup is rather simple: just the Perceval Python module needs to be installed.
 This is the case explained in sections
-[Retrieving git metadata](../perceval/git.html), [Retrieving data from GitHub repositories](../perceval/github.html) and
-[Mail archives](../perceval/mail.html).
+[Retrieving git metadata](../perceval/git.md), [Retrieving data from GitHub repositories](../perceval/github.md) and
+[Mail archives](../perceval/mail.md).
```

Comment:
I think you might need to revert these changes. The docs are actually generated using Jekyll and it is accessible at [chaoss.github.io/grimoirelab-tutorial](https://chaoss.github.io/grimoirelab-tutorial/). The links work fine just the way they are now.

Reply:
Okay Sir , 
but "chaoss.github.io/grimoirelab-tutorial " this is hosted on Github Pages and on this repository everything is in .md format and Github pages serves their repository live content , then how and when it is converting to .html from .md extension using Jekyll .
It can be a silly question , but i got confused in this .

Reply:
Jekyll converts these pages from md to html files and they are served.

For example, https://github.com/chaoss/grimoirelab-tutorial/blob/master/python/scripting.md is converted to https://chaoss.github.io/grimoirelab-tutorial/python/scripting.html

Also, you need not address me as sir :slightly_smiling_face:

Reply:
Now I understand and also readed some documents related to Jekyll  .

Reply:
And i did everything as per your Instructions , to make this PR ready to merge . If anything left please tell me i will do that .

Reply:
I think you have missed this https://github.com/chaoss/grimoirelab-tutorial/pull/153#discussion_r598103536
Let me know if it is not clear.

Reply:
Ohh , sorry i missed that one . 
Give me a second  to let me fix that